src/test/java/com/openmrs/migrator/core/services/BootstrapServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package com.openmrs.migrator.core.services;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class BootstrapServiceTest {
+
+  @Autowired private BootstrapService bootstrapService;
+
+  private List<String> folders = Arrays.asList("folder0", "folder1", "folder2");
+
+  private Path file = Paths.get("file.unknown");
+
+  @Test
+  public void createDirectoryStructureSuccess() throws IOException {
+
+    removeFolders();
+
+    int numberOfCreatrions = bootstrapService.createDirectoryStructure(folders, file);
+    assertEquals(4, numberOfCreatrions);
```

Comment:
I think you should test that the right folders got created here, not just that the correct number of folders were created

Reply:
done